### ISSUE TITLE
Collaboration project proposal

### ISSUE THREAD

**Issue by SomeDevWeb** [simondevyoutube/Quick_3D_MMORPG#15]
Dear Simon (and any developer that comes across this post),
 
I'm a web dev and I watched your threejs MMO youtube video which I think it's awesome! 
For a very long time, I have been interested in building or joining a team that develops a Threejs-based MMO...
A couple of years ago I discovered one of the most gorgeous games, the Ironbane project. It was for a long time in the hall of fame on google projects page, but for some reason a few years ago it just died out and its development just stopped. It still has all the code base on github and last year I tried it and I could install it and it definitely looks awesome. 

Now, in the recent months, I feel a clear and strong urge to try to revive this awesome project and a couple of weeks ago I actually tried to install it and very surprisingly it worked after a few hiccups obviously. 
So I ve set it and left it running on a VPS and I also created a website post where describe in more detail my idea of the New Game Concept. 
Here the future dev. team will be able to interact and gather data, share info and work together. 
URL: https://www.webcreative.me/three-js-mmorpg-game-concept-ironbane/

The game doesn't have a website anyore, but the old dev team still has a dedicated discord channel where I am a member too and which I will disclose to anyone interested.

So, my proposal to all of you is to JOIN OUR FORCES and try to revive this project and make it a mighty game!!!

In reality, even though the game fully works now and it has a lot of the basic structure made, the code base is quite old and I obsolete. Also, after I spoke with a couple of main devs, they said they would strongly suggest to almost start from scratch if we want to revive it. Of course we have tons of things still keep and use, but probably a clean slate would be an idea to think about and discuss...

In any case, I am absolutely convinced that the business/collaboration model I thought of is a good engine for us to move forward. 
So please have a look at the page above and leave some feedback either there or here.
	
As a thought, the Ironbane name is what they chose, but it doesn't even have to be the same for the new game. Personally, I think it sounds cool, but it's open to discussion.
	
In conclusion, I really hope and pray you have enough interest and bit of time to join us. 
I am looking forward to your comments, whichever they are. 
Thanks for your time. 
Andy

**Comment by rookieKing** [comment]
https://github.com/rookieKing/Quick_3D_MMORPG Here is my fixed version.
If you're still building a team, I'd like to join, I'm also a web developer.
The discord invitation above has expired.

**Comment by Akul2010** [comment]
Hey hi,

I always wanted to create something of this sort as well. I fixed a few bugs in his code and made a pull request for it. (https://github.com/simondevyoutube/Quick_3D_MMORPG/pull/17#issue-1456738608) Hopefully, he pulls it, because it won't run on modern browsers otherwise. Anyway, I can only work on it in my free time, but can I help with the project? I am also quite experienced with blender, so I can help with code _and_ assets (but again, only in my free time)
